GeneralsMD/Code/Tools/W3DView/Utils.cpp

```diff
@@ -889,20 +757,6 @@ Load_RC_Texture (LPCTSTR resource_name)
 	unsigned char *res_data = resource_file.Peek_Data ();
 	unsigned int data_size = resource_file.Size ();
 
-	//
-	//	Create a texture from the raw image data
-	//
-#ifdef WW3D_DX8
```

Comment:
Keep the EA comment above and write that this is unimplemented.

`// TheSuperHackers @info Not implemented`